src/routes/Project/components/Table.js

```diff
@@ -1,11 +1,16 @@
-import React from 'react'
+import React from 'react';
 
 export const Table = (props) => (
```

Comment:
also, how can the Table access the store? I'm not quite sure i understand that part.